packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -276,20 +276,24 @@ There are number of new environment variables, and you may use this table as a r
 
 ### Create Accounts from the `ReleaseGold` contracts
 
-In order to use the balances from `ReleaseGold` contracts, we need to create associated Accounts. In the Baklava network, you can look up your Beneficiary address in [the published mapping](https://gist.githubusercontent.com/nategraf/a87f9c2e488ab2d38a0a3c09f5d4ca2b/raw) to find your `ReleaseGold` contract addresses. If you are a genesis validator, your two Beneficary addresses will be the provided `CELO_VALIDATOR_ADDRESS` and `CELO_VALIDATOR_GROUP_ADDRESS`.
+In order to participate on the network (lock gold, vote, validate) from a `ReleaseGold` contract, we need to create an Account at the address of the `ReleaseGold` contract. In the Baklava network, you can look up your Beneficiary address in [the published mapping](https://gist.githubusercontent.com/nategraf/a87f9c2e488ab2d38a0a3c09f5d4ca2b/raw) to find your `ReleaseGold` contract addresses. If you are a genesis validator, your two Beneficary addresses will be the provided `CELO_VALIDATOR_BENEFICIARY_ADDRESS` and `CELO_VALIDATOR_GROUP_BENEFICIARY_ADDRESS`.
+
+{% hint style="info" %}
+An earlier version of this document set the beneficiary addresses to `CELO_VALIDATOR_ADDRESS` and `CELO_VALIDATOR_GROUP_ADDRESS`, but the attestation service expects these to refer to the `ReleaseGold` contract, so please move your beneficiary addresses to reflect the environment variables above.
```

Comment:
@bohan-zhang PTAL and critique

Reply:
Hmm I worry this could be confusing to some folks, can we be more explicit about which ENV vars here need to be set to which values?